src/test/suite/recorded.test.ts

```diff
@@ -75,6 +76,8 @@ async function runTest(file: string) {
     fixture.initialState.documentContents,
     fixture.languageId
   );
+  // Override any user defaults, all tests are recorded as tabSize = 4
+  editor.options.tabSize = DEFAULT_TAB_SIZE_FOR_TESTS;
```

Comment:
I think we also want to force soft tabs too. I have `"editor.insertSpaces": false` and all my snapshots changed 😅